destination/simulator_test.go

```diff
@@ -8,14 +8,14 @@ import (
 
 func Test_GivenDestinationIsCorrect_WhenSimulatorIsCreated_ThenItReturnsCorrectSimulator(t *testing.T) {
 	// Given
-	destination := "platform=iOS Simulator,name=iPhone 8 Plus,OS=latest"
+	destination := "platform=iOS Simulator,name=iPhone 8 Plus,OS=latest,arch=x86_64"
```

Comment:
I'd rather create a new test case for the arch flag to keep testing the no-arch-flag case as well.

Reply:
added new test